lib/x12.rb

```diff
@@ -1,6 +1,25 @@
+module X12
+  ROOT = File.expand_path('../..', __FILE__)
+  # EMPTY = Empty.new()
+  TEST_REPEAT = 100
+end
+
 require 'nokogiri'
 
-require "x12/version"
+require 'x12/version'
+require 'x12/attributes'
+require 'x12/structures/base'
+require 'x12/structures/field'
+require 'x12/structures/segment'
+require 'x12/structures/loop'
+require 'x12/templates/base'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.